doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
+
+The IoT Edge Hub module exposes a number of metrics in the Prometheus exposition format that provide insights into its operational state.
+
+### How to enable
+
+As of release 1.0.9, metrics are exposed as an experimental feature available at http port **9600** of the Edge Hub module. To enable, the following environment variables should be set for the module (make note of the double underscores):
+
+| Environment Variable Name                | value  |
+|------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`          | `true` |
+| `ExperimentalFeatures__EnableMetrics`    | `true` |
+
+### Metrics
+
+| Name                                                        | Dimensions                                                                                                                                                                           | Description                                                                                                                                                                                              | Type    |
+|-------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `edgehub_gettwin_total`                                     | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `source` (Operation source)<br> `id` (Module ID)                                                                | Get Twin calls                                                                                                                                                                                           | counter |
+| `edgehub_messages_received_total`                           | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br>  `protocol` (Protocol received on)<br> `id` (Module ID)                                                         | Messages received from clients                                                                                                                                                                           | counter |
+| `edgehub_messages_sent_total`                               | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `from` (Message source)<br> `to` (Message destination)                                                          | Messages sent to clients/upstream                                                                                                                                                                        | counter |
```

Comment:
Nit - description description: Total number of messages sent to a module or upstream

Reply:
@varunpuranik Does this not include C2D messages to downstream devices?